pkg/cmd/grafana-cli/services/io_util.go

```diff
@@ -21,5 +22,5 @@ func (i IoUtilImp) ReadDir(path string) ([]os.FileInfo, error) {
 }
 
 func (i IoUtilImp) ReadFile(filename string) ([]byte, error) {
-	return ioutil.ReadFile(filename)
+	return ioutil.ReadFile(filepath.Clean(filename))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.